ownCloud/Settings/MediaUploadSettingsSection.swift

```diff
@@ -44,12 +54,85 @@ extension UserDefaults {
 			return self.bool(forKey: MediaUploadKeys.ConvertVideosToMP4Key.rawValue)
 		}
 	}
+
+	public var instantUploadPhotos: Bool {
+		set {
+			self.set(newValue, forKey: MediaUploadKeys.InstantUploadPhotosKey.rawValue)
+		}
+
+		get {
+			return self.bool(forKey: MediaUploadKeys.InstantUploadPhotosKey.rawValue)
+		}
+	}
+
+	public var instantUploadVideos: Bool {
+		set {
+			self.set(newValue, forKey: MediaUploadKeys.InstantUploadVideosKey.rawValue)
+		}
+
+		get {
+			return self.bool(forKey: MediaUploadKeys.InstantUploadVideosKey.rawValue)
+		}
+	}
+
+	public var instantUploadBookmarkUUID: UUID? {
+		set {
+			self.set(newValue?.uuidString, forKey: MediaUploadKeys.InstantUploadBookmarkUUIDKey.rawValue)
+		}
+
+		get {
+			if let uuidString = self.string(forKey: MediaUploadKeys.InstantUploadBookmarkUUIDKey.rawValue) {
+				return UUID(uuidString: uuidString)
+			} else {
+				return nil
+			}
+		}
+	}
+
+	public var instantUploadPath: String? {
+
+		set {
+			self.set(newValue, forKey: MediaUploadKeys.InstantUploadPathKey.rawValue)
+		}
+
+		get {
+			return self.string(forKey: MediaUploadKeys.InstantUploadPathKey.rawValue)
+		}
+	}
+
+	public var instantUploadPhotosAfter: Date? {
+		set {
+			self.set(newValue, forKey: MediaUploadKeys.InstantUploadPhotosAfterDateKey.rawValue)
+		}
+
+		get {
+			return self.value(forKey: MediaUploadKeys.InstantUploadPhotosAfterDateKey.rawValue) as? Date
+		}
+	}
+
+	public var instantUploaVideosAfter: Date? {
```

Comment:
This should be `instantUploadVideosAfter`.

Reply:
@felix-schwarz fixed